ManagedClientConsoleAppSample/Program.cs

```diff
@@ -78,6 +80,7 @@ private static void ListProjects(AuthenticationHeaderValue authHeader)
                 {
                     Console.WriteLine("\tSuccesful REST call");
                     Console.WriteLine(response.Content.ReadAsStringAsync().Result);
+                    Thread.Sleep(10000);
```

Comment:
I just felt that since the console closes immediately at the end of the execution it would be hard for a user to validate if the sample works or not. I realize that it doesn't have a place in a code sample, but maybe with comments around it stating it is unnecessary it could be okay? I am totally flexible here, if you think the above isn't a valid concern for users I understand.